tests/test_tiebreaks.py

```diff
@@ -0,0 +1,1148 @@
+
+    def test_played_with_black(self):
+        assert {
+            player.id: individual.games_played_with_black(player, self.tournament)
+            for player in self.tournament.players_by_id.values()
+        } == {
+            2: 3,
+            1: 2,
+            3: 2,
+            4: 2,
+            16: 2,
+            6: 2,
+            5: 2,
+            8: 2,
+            11: 2,
+            15: 3,
+            14: 2,
+            12: 0,
+            7: 3,
+            13: 3,
+            9: 1,
+            10: 3,
+        }
+
+    def test_won_with_black(self):
+        assert {
+            player.id: individual.games_won_with_black(player, self.tournament)
+            for player in self.tournament.players_by_id.values()
+        } == {
+            2: 1,
+            1: 1,
+            3: 1,
+            4: 1,
+            16: 1,
+            6: 1,
+            5: 0,
+            8: 0,
+            11: 0,
+            12: 0,
+            14: 1,
+            15: 1,
+            7: 0,
+            9: 0,
+            13: 1,
+            10: 1,
+        }
+
+    def test_games_elected_to_play(self):
+        assert {
+            player.id: individual.rounds_elected_to_play(player, self.tournament)
+            for player in self.tournament.players_by_id.values()
+        } == {
+            2: 5,
+            1: 5,
+            3: 5,
+            16: 5,
+            4: 4,
+            6: 5,
+            5: 5,
+            8: 5,
+            11: 5,
+            15: 5,
+            12: 3,
+            14: 3,
+            7: 5,
+            13: 5,
+            9: 3,
+            10: 5,
+        }
+
+    def test_progressive_scores(self):
+        assert {
+            player.id: individual.progressive_scores(player, self.tournament)
+            for player in self.tournament.players_by_id.values()
+        } == {
+            2: 13,
+            4: 11.5,
+            1: 11,
+            3: 11,
+            16: 10.5,
+            6: 6,
+            8: 8.5,
+            11: 5.5,
+            5: 5,
+            12: 7,
+            15: 7,
+            14: 6,
+            13: 7,
+            7: 6,
+            9: 2.5,
+            10: 4,
+        }
+
+    def test_progressive_cut1(self):
+        assert {
+            player.id: individual.progressive_scores(player, self.tournament, cut=1)
+            for player in self.tournament.players_by_id.values()
+        } == {
+            2: 12,
+            3: 10.5,
+            4: 10.5,
+            1: 10,
+            16: 10,
+            6: 6,
+            8: 8,
+            5: 5,
+            11: 5,
+            12: 7,
+            15: 7,
+            14: 5,
+            13: 6,
+            7: 5,
+            9: 2.5,
+            10: 4,
+        }
+
+    def test_buchholz(self):
+        assert {
+            i: individual.buchholz(
+                self.tournament.players_by_id[i],
+                self.tournament,
+            )
+            for i in (2, 1, 3, 5, 8, 11, 16, 4)
```

Comment:
Why aren't all players tested.  The table on page 32 of the PDF gives the results of each tiebreak for each player